node_modules/.svelte2tsx-language-server-files/svelte-native-jsx.d.ts

```diff
@@ -0,0 +1,32 @@
+declare namespace svelteNative.JSX {
```

Comment:
let's remove this again, and probably gitignore it.
I also have this randomly pop up sometimes, it's auto generated, I'm not sure why.